src/test/rustdoc-json/enum_variant_hidden.rs

```diff
@@ -0,0 +1,10 @@
+#![no_std]
```

Comment:
Some stylistic nitpicks (feel free to ignore)

- I don't find it's worth using `#![no_std]`, as it doesnt remove enought to make inspecing the output plesent. (`wc -c` for empty rust file: 371,664 for nothing, 97,448 for `#[no_std]`, 478 for #![no_core]`
- For json, only the first selection needs to use a the path to a json file, the rest can just use `-` (Also, we should change jsondocck to know the name of the json file)
- A link to the issue would be convenient for future reference.

Reply:
All very good points. I'll apply them all. Can you open an issue for jsondocck improvement idea please?